e2e/test/scenarios/models/models.cy.spec.js

```diff
@@ -524,10 +524,10 @@ describe("scenarios > models", () => {
     turnIntoModel();
 
     visitCollection("root");
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Models");
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("A model");
+    cy.findByTestId("pinned-items").within(() => {
```

Comment:
These tests seemed to be passing before because we have `Models` on the sidebar rather than the header of `Models` in the pinned items section of the collection page